src/middlewares/superadminAuth.js

```diff
@@ -0,0 +1,17 @@
+class superadminAuth {
+  static verifyToken = async (req, res, next) => {
```

Comment:
the naming here can be something else like checkAdmin/verifyAdmin but not just verifyToken